src/SFML/Audio/ALCheck.cpp

```diff
@@ -42,14 +42,13 @@ namespace sf
 namespace priv
 {
 ////////////////////////////////////////////////////////////
-void alCheckError(const char* file, unsigned int line, const char* expression)
+void alCheckError(const std::filesystem::path& file, unsigned int line, const char* expression)
```

Comment:
Does this work as expected? Why didn't we use `std::string` on the function directly in the past? 🤔

Reply:
As far as I can tell it's fine. No idea why we didn't use `std::string` before.